pkg/gardenlet/controller/shoot/shoot_control_reconcile.go

```diff
@@ -347,6 +350,23 @@ func (c *Controller) runReconcileShootFlow(o *operation.Operation) *gardencorev1
 			Fn:           flow.TaskFn(botanist.WaitUntilTunnelConnectionExists).SkipIf(o.Shoot.HibernationEnabled),
 			Dependencies: flow.NewTaskIDs(deployManagedResources, waitUntilNetworkIsReady, waitUntilWorkerReady),
 		})
+		_ = g.Add(flow.Task{
+			Name: "Migrating Kubernetes API server to ClusterIP service in the Seed cluster",
```

Comment:
```suggestion
			Name: "Migrating Kubernetes API server service to ClusterIP service in the Seed cluster if necessary",
```

Reply:
The comment is only valid if sniPhase=enabling . If sniPhase=disabling then the svc type is changed to LoadBalancer. 
Can we find a more generic description here?